recipe/meta.yaml

```
@@ -1,43 +1,67 @@
 {% set name = "mistralai" %}
-{% set version = "0.4.2" %}
+{% set version = "1.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mistralai-{{ version }}.tar.gz
-  sha256: 5eb656710517168ae053f9847b0bb7f617eda07f1f93f946ad6c91a4d407fd93
+  sha256: 94e3eb23c1d3ed398a95352062fd8c92993cc3754ed18e9a35b60aa3db0bd103
+  patches:
+    # Switch from poetry-core to hatchling to work with [project] section metadata upstream
+    - patches/0001-fix-build-system.patch
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: True  # [py<310]
```

```suggestion
  skip: True  # [py<39]
```
https://github.com/mistralai/client-python/blob/v1.7.0/pyproject.toml#L7